r/vignettes/install.Rmd

```diff
@@ -154,15 +154,15 @@ knitr::include_graphics("./r_source_libarrow_source.png")
 ```
 
 The "less easy" way to install arrow is to install both the R package and the underlying Arrow C++ library (libarrow) from source. This method is somewhat more
-difficult because compiling and installing R packages with C++ dependencies 
-generally requires installing system packages, which you may not have privileges 
-to do, and/or building the C++ dependencies separately, which introduces all sorts 
+difficult because compiling and installing R packages with C++ dependencies
+generally requires installing system packages, which you may not have privileges
+to do, and/or building the C++ dependencies separately, which introduces all sorts
```

Comment:
Lots of extra whitespace. I can back it out, but we'll be fighting against it with anyone who uses vscode on this so I think we should just merge it here.